sections/scaffold-system.md

```diff
@@ -1,43 +1,93 @@
 
 * `aquifer create --help` - Returns helpful documentation for this command.
-* `aquifer create my_project_name` - Creates a folder called `my_project_name` in the current working directory, and scaffolds a drush make Drupal site root.
+* `aquifer create my_project_name` - Creates a folder called `my_project_name` in the current working directory, and scaffolds a drush make, Drupal 7 site root.
+* `aquifer create my_project_name -d 8` - Creates a folder called `my_project_name` in the current working directory, and scaffolds a drush make, Drupal 8 site root.
 * `aquifer create my_project_name --config_file /path/to/config.json` - Creates a folder called `my_project_name` in the current working directory, and uses the `config.json` to determine the file structure that should be scaffolded.
 
 ### Using a config file
+
 As mentioned previously, `aquifer create` accepts a `--config_file` or `-c` flag that takes a path to a config JSON file.
 
-When supplied, this JSON file will be used to scaffold the project. File and folder names will be created based on what is specified in the passed-in file, allowing one to have complete control over the source tree structure of the entire project. Here's an example `config.json` file:
+When supplied, this JSON file is used to scaffold the project. File and folder names will are created based on what is specified in the passed-in file. This allows one to have complete control over the source tree structure of the entire project. Here's an example `config.json` file:
 
 ```javascript
 {
   "name": false,
-  "paths": {
-    "make": "drupal.make.yml",
-    "lock": false,
-    "settings": "settings",
-    "build": "build",
-    "root": "root",
-    "themes": {
-      "root": "themes",
-      "contrib": "themes/contrib",
-      "custom": "themes/custom"
-    },
-    "modules": {
-      "root": "modules",
-      "contrib": "modules/contrib",
-      "custom": "modules/custom",
-      "features": "modules/features"
+  "core": 7,
+  "build": {
+    "method": "drush make",
+    "directory": "build",
+    "makeFile": "drupal.make.yml"
+  },
+  "sync": {
+    "directories": {
+      "root": {
+        "destination": "",
+        "method": "symlink",
+        "conflict": "overwrite"
+      },
+      "modules/custom": {
+        "destination": "sites/all/modules/custom",
+        "method": "symlink",
+        "conflict": "overwrite"
+      },
+      "modules/features": {
+        "destination": "sites/all/modules/features",
+        "method": "symlink",
+        "conflict": "overwrite"
+      },
+      "themes/custom": {
+        "destination": "sites/all/themes/custom",
+        "method": "symlink",
+        "conflict": "overwrite"
+      },
+      "files": {
+        "destination": "sites/default/files",
+        "method": "symlink",
+        "conflict": "overwrite"
+      }
```

Comment:
These changes look more like the generated `aquifer.json` than `config.json`. Also, as `aquifer.json`, it doesn't work any more. The "sync" key should not have an extra level for "directories" and "files", but just list everything.